src/features/island/buildings/components/building/smoothieShack/SmoothieShackModal.tsx

```diff
@@ -49,9 +49,9 @@ export const SmoothieShackModal: React.FC<Props> = ({
       <Panel
         bumpkinParts={{
           body: "Light Brown Farmer Potion",
-          hair: "Explorer Hair",
-          pants: "Lumberjack Overalls",
-          shirt: "Blue Farmer Shirt",
+          hair: "Brown Long Hair",
+          pants: "Farmer Pants",
+          shirt: "Pineapple Shirt",
           tool: "Golden Spatula",
```

Comment:
```suggestion
          tool: "Parsnip",
```

I reckon give old mate a parsnip 😁 

<img width="579" alt="parsnip" src="https://user-images.githubusercontent.com/41215134/210679184-ab7ce77a-0f97-4371-8ad1-2cac26950a7a.png">